src/views/GovernanceOverview/index.tsx

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import { withRouter, RouteComponentProps } from 'react-router-dom';
+import Header from 'components/Header';
+import './index.scss';
+import CollapsableLink from 'components/shared/CollapsableLink';
+import Button from 'components/shared/Button';
+
+type NumberedListItemProps = {
+  index: string;
+  header: string;
+  body: string;
+};
+
+const NumberedListContinuingItem = ({
+  index,
```

Comment:
🎨  Since index means something very specific from a programming standpoint, it might be better to name this `stepNumber` or something similar. For example, I was surprised that it was text (not a number), and that you didn't have to do any index max on it to convert to a human-readable number.